ebos/ecloutputblackoilmodule.hh

```diff
@@ -1866,6 +1866,10 @@ public:
     const std::map<std::pair<std::string, int>, double>& getBlockData()
     { return blockData_; }
 
+    const Opm::Inplace initialInplace() const {
```

Comment:
Is this function supposed to return a reference to the contained object or a copy of that object?  In the latter case, it should typically not return a `const` object.

Reply:
Ahhh - `&` here and a `&` there; it should be a const ref. Will fix